src/qt/clientmodel.cpp

```diff
@@ -35,6 +35,7 @@ ClientModel::ClientModel(interfaces::Node& node, OptionsModel *_optionsModel, QO
 {
     cachedBestHeaderHeight = -1;
     cachedBestHeaderTime = -1;
+    m_cached_tip_blocks = uint256();
```

Comment:
6002d1185db89af124a257c894195a7f1b04c1f2

Mind using a default member initializer:
```
    uint256 m_cached_tip_blocks GUARDED_BY(m_cached_tip_mutex) {};
```

?

Reply:
sure np. Can move the other members initializations (cachedBestHeaderXXX) if that is needed in another commit too.